community/netsurf/build

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -e
+
+make
+make DESTDIR="$1" install
```

Comment:
No need to set `--prefix=/usr` with `./configure`?

Reply:
Correct. This netsurf is very purist in the way they do things, if you haven't tried it btw, you might be impressed!

Reply:
actually wait now im confused too

Reply:
i have it installed, looks like it's in /usr/bin but i don't know which route it took, e.g. did it get installed through the symlink or proper?

Reply:
```sh
make \
    LIBDIR=lib \
    INCLUDEDIR=include \
    PREFIX=/usr

make \
    LIBDIR=lib \
    INCLUDEDIR=include \
    PREFIX=/usr \
    install
```

Reply:
Also, enable GTK3 as GTK2 will be dropped from KISS once Firefox drops it.

```
    TARGET=gtk \
    NETSURF_GTK_MAJOR=3 \
```

Reply:
okay, thought gtk2 might be more forward looking (lol)

Reply:
@dylanaraps what about DESTDIR